src/libs/karm-math/trans.h

```diff
@@ -12,6 +12,9 @@ template <typename T>
 union Trans2 {
     using Scalar = T;
 
+    static constexpr const T ZERO = T{};
+    static constexpr const T ONE = T{1};
+
```

Comment:
we can get fancy and use algebra terms https://en.wikipedia.org/wiki/Matrix_multiplication#Abstract_algebra